etc/openstack_deploy/group_vars/all/osa.yml

```diff
@@ -96,3 +96,14 @@ haproxy_extra_services:
 
 # Define the distro version globally
 repo_build_os_distro_version: "{{ (ansible_distribution | lower) | replace(' ', '_') }}-{{ ansible_distribution_version.split('.')[:2] | join('.') }}-{{ ansible_architecture | lower }}"
+
+# remove once https://review.openstack.org/#/c/573822/ merges
+ironic_ironic_conf_overrides:
+  glance:
+    auth_url: "http://{{ internal_lb_vip_address }}:35357/v3"
+    auth_type: "password"
```

Comment:
like the upstream review, most of these things should be variables, not hard coded. (domain, project, user, auth_type)